
<launch>

    <group ns="sensors">

    <node name="combustion_transducer" pkg="sensors"
          type="pressure_transducer_driver.py" output="screen"
          args="AIN3">
    </node>

    </group>

</launch>
